tasks/main.yml

```diff
@@ -57,7 +71,7 @@
 - name: Download Caddy
   get_url:
     url: "{{ caddy_url }}"
-    dest: "{{ caddy_home }}/caddy.tar.gz"
+    dest: "{{ caddy_home }}/{{ caddy_bin_name }}"
```

Comment:
As far as I can see you always update caddy now? I do not think that `caddy_update: false` works anymore.

Reply:
I think that the behaviour here is still correct but feel free to point out the error in my thinking. My understanding is this:

- 'Get all Caddy releases' and 'Set Caddy features' both have `when: caddy_update`
- If neither of there happen then `caddy_releases_cache` and `caddy_features_cache` will both report not changed
- The first 'Download Caddy' step has `when: caddy_releases_cache.changed or caddy_features_cache.changed` so will not be executed in this situation
- The second 'Download Caddy' step does not specify the `force` parameter [meaning it defaults to false](https://docs.ansible.com/ansible/latest/modules/get_url_module.html#parameter-force) and will therefore download the file only if it doesn't already exist

This means that if you have `caddy_update: false` the download will only be done if the file doesn't already exist.